### PR TITLE
Disable Sql Server tests on windows 7 & windows server 2008

### PR DESCRIPTION
**Pull request by smitpatel**
@ryanbrandenburg

### CODE REVIEW

File: test/EFCore.Benchmarks.EFCore/EFCore.Benchmarks.EFCore.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\src\EFCore.Specification.Tests\EFCore.Specification.Tests.csproj" />
```
Comment (smitpatel) [inline test/EFCore.Benchmarks.EFCore/EFCore.Benchmarks.EFCore.csproj]: to import the attribute

File: src/EFCore.Specification.Tests/TestUtilities/Xunit/WindowsVersions.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.EntityFrameworkCore.Specification.Tests.TestUtilities.Xunit
+{
+    public static class WindowsVersions
+    {
+        public const string Win7 = "6.1";
+
+        public const string Win2008R2 = Win7;
```
Comment (ryanbrandenburg) [inline src/EFCore.Specification.Tests/TestUtilities/Xunit/WindowsVersions.cs]: These look backwards?

**Comment by smitpatel** [comment]
filed #8178

File: test/EFCore.Benchmarks.EFCore/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -1,6 +1,11 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.EntityFrameworkCore.Specification.Tests.TestUtilities.Xunit;
 using Xunit;
 
 [assembly: CollectionBehavior(DisableTestParallelization = true)]
+
+#if NETCOREAPP2_0
+[assembly: OSSkipCondition(TestPlatform.Windows, WindowsVersions.Win2008R2, WindowsVersions.Win7, SkipReason = "SqlClient has issue with .netcoreapp2.0 on win7.")]
```
Comment (ryanbrandenburg) [inline test/EFCore.Benchmarks.EFCore/Properties/AssemblyInfo.cs]: Just so I'm clear, no tests in EFCore.Benchmarks.EFCore will be run on win2008R2 or Win7?

File: src/EFCore.Specification.Tests/TestUtilities/Xunit/WindowsVersions.cs (hunk 2 above)
Comment (smitpatel) [inline src/EFCore.Specification.Tests/TestUtilities/Xunit/WindowsVersions.cs]: code copied from Testing repo

**Comment by ryanbrandenburg** [comment]
:shipit: if the build passes.

**Comment by smitpatel** [comment]
@ryanbrandenburg 
> Some questions about what exactly it is we're doing here but  if it unblocks the build

There is issue in underlying SqlClient on win7 which is failing all SqlServer tests for EF. We are disabling all the tests which are using SqlServer. Since windows specific skip condition was not part of EF code base & we don't plan to take dependency on Testing, I added that specific attribute for this case and disabled tests for netcoreapp2.0 and on specific windows versions.

**Pull request merged**